api/processors/actions/__init__.py

```diff
@@ -16,4 +18,7 @@
     "StudentTransferProcessor",
     "StudentMissedClassProcessor",
     "TeacherTransferProcessor",
+    "StudentWantOnLeave",
```

Comment:
```suggestion
    "StudentWentOnLeave",
```
or maybe something completely different should be here (or nothing?)

Reply:
Ah, it is just redundant default import, it is not needed here